### DISCUSSION TITLE
Drive Stats averages vs' Mean calculations?

### DISCUSSION THREAD

**Discussion by timmills** [teslamate-org/teslamate#3883] [General]
How are the drive stats calculated (average day / month etc as they dont seem to be correct)
in the last 12 months 27085 km yet average driven per day is 37.9 km and the average drive is 1.5 km per day with 6.83 kWh per day

**Comment by eliluong** [comment]
It is currently calculating median instead of mean, so the label is incorrect or the formula has not been updated.
